### ISSUE TITLE
Save & Share URL shortener fails with multiple plugins

### ISSUE THREAD

**Issue by kdeloach** [CoastalResilienceNetwork/GeositeFramework#651]
If you activate a few plugins and click Save & Share, it will occasionally fail to shorten the URL.

There's a possibility that the payload is too large.

The response back from Google is:

```
{
 "error": {
  "errors": [
   {
    "domain": "global",
    "reason": "invalid",
    "message": "Invalid Value",
    "locationType": "parameter",
    "location": "resource.longUrl"
   }
  ],
  "code": 400,
  "message": "Invalid Value"
 }
}
```

Tested on the NJ Dev site.

**Comment by caseypt** [comment]
I tested this out a bit, and it appears to fail when the source URL is longer than ~4,000 characters. I tried one of the failing URLs on bit.ly, and it also failed with a similar message ("Invalid value" vs "Invalid URL").

When the shortener fails however, the unshortened URL is displayed, which seems like a graceful fallback.

Are we interested in somehow otherwise alerting the user that we can't shorten the app URL if there are too many plugins turned on?
